src/robot/htmldata/rebot/testdata.js

```diff
@@ -156,15 +156,25 @@ window.testdata = function () {
         }
         return metadata;
     }
-
+    
+    /*  This function gets the execution status results from the suites ran and calculates the total
+        passed, failed and skippped for critical and non critical tests. 
+        Stat[0] = Total tests ran
+        Stat[1] = Total tests passed
+        Stat[2] = Total tests skipped
+        stat[3] = Total critical tests ran
+        stat[4] = Total critical tests passed
+        stat[5] = Total critical tests failed
```

Comment:
There is inconsistency in this comments as it related to total and critical total.  For total, you are collecting the total ran, passed and *skipped*.  However, for critical total, you're collecting total ran, passed and *failed*.

Is this just a documentation issue?